src/templates/components/CommunityTemplateInstallOverlay.tsx

```diff
@@ -150,6 +150,11 @@ class CommunityTemplateInstallOverlayUnconnected extends PureComponent<Props> {
 
       this.props.getBuckets()
       this.props.notify(communityTemplateInstallSucceeded(templateDetails.name))
+      event(
+        'community_template.' +
+          normalizeEventName(templateDetails.name) +
+          '.installed'
+      )
```

Comment:
```suggestion
      event(`community_template.${normalizeEventName(templateDetails.name)}.installed`)
```

prefer template strings